utils/perl5lib/Batch/BatchMaker.pm

```diff
@@ -812,7 +812,7 @@ sub setCESMRun()
         }
     }
 
-    $self->{'mpirun'} .= "qx( " . $executableString . " " . $mpiargstring .  " " . $defaultrunsuffix . ");";
+    $self->{'mpirun'} .= "run_cmd( " . $executableString . " " . $mpiargstring .  " " . $defaultrunsuffix . ")";
```

Comment:
I think that the run_cmd should be in template.case.run and not here.   In fact that is the way you are using it at line 142 of template.case.run